keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ApiVersion.kt

```diff
@@ -0,0 +1,5 @@
+package com.netflix.spinnaker.keel.api
+
+data class ApiVersion(val group: String, val version: String = "1") {
+  fun qualify(kind: String) = ResourceKind(group, kind, version)
+}
```

Comment:
I recreated `ApiVersion` so we have a way of defining a constant that we can easily use to create `ResourceKind` instances within that API version.